src/item/Project.js

```diff
@@ -888,7 +888,11 @@ var Project = PaperScopeItem.extend(/** @lends Project# */{
                 size = this._scope.settings.handleSize,
                 version = this._updateVersion;
             for (var id in items) {
-                items[id]._drawSelection(ctx, matrix, size, items, version);
+                var item = items[id];
+                // Do not draw selection if item is used as symbol definition.
+                if (!item._symbol) {
+                    item._drawSelection(ctx, matrix, size, items, version);
+                }
```

Comment:
I'm not sure this is the right place to handle this. Shouldn't `Item#setSelected()` not select the item when it's used as a symbol definition?

Reply:
Oh, yes, this could also be a solution, I pushed a new commit including it.